lottie-lite.php

```diff
@@ -129,11 +129,13 @@
  */
 add_filter( 'upload_mimes', function( $mime_types ) {
 	$mime_types['json'] = 'text/plain'; // This is needed to allow uploading the file type.
+	$mime_types['json'] = 'application/json'; // This is needed to allow uploading the file type.
```

Comment:
This will just override the line before it and break it for others.

Reply:
That's going to be a problem, then, since https://developer.wordpress.org/reference/functions/wp_get_mime_types/ doesn't seem to offer a syntax for this array that would allow multiple MIME types to share the same extension.